forloop_modules/globals/variable_handler.py

```diff
@@ -196,6 +196,8 @@ def get_db_connection(self, db_name):
 # api_variable_handler = VariableHandler()
 variable_handler = LocalVariableHandler()
 
+
+# TODO: Disable ASAP - using global lists and dicts to store values is just wrong
```

Comment:
This remark makes sense only in OOP, I dont think it's wrong, it just doesnt impose rigid structure on the data stored in these objects